stories/button.stories.ts

```diff
@@ -3,46 +3,41 @@ import '../igniteui-webcomponents.js';
 import { Story } from './story.js';
 import { ifDefined } from 'lit-html/directives/if-defined';
 
+// region default
 export default {
   title: 'Button',
   component: 'igc-button',
   argTypes: {
-    disabled: {
-      control: 'boolean',
-      description: 'Determines whether the button is disabled.',
-      table: {
-        type: {
-          summary: 'boolean',
-        },
-        defaultValue: {
-          summary: 'false',
-        },
-      },
-    },
-    size: {
+    type: {
+      description: 'The type of the button. Defaults to undefined.',
       control: {
         type: 'inline-radio',
-        options: ['small', 'medium', 'large'],
+        options: ['button', 'reset', 'submit'],
       },
-      defaultValue: 'large',
+    },
+    disabled: {
+      description: 'Determines whether the button is disabled.',
+      defaultValue: 'false',
+      control: 'boolean',
     },
     variant: {
+      defaultValue: 'flat',
       control: {
         type: 'inline-radio',
         options: ['flat', 'raised', 'outlined', 'fab'],
       },
-      defaultValue: 'flat',
     },
-    type: {
+    size: {
+      description: 'Determines the size of the component.',
+      defaultValue: 'large',
       control: {
         type: 'inline-radio',
-        options: ['button', 'reset', 'submit'],
+        options: ['small', 'medium', 'large'],
       },
-      defaultValue: 'button',
     },
   },
 };
-
+// endregion
 interface ArgTypes {
```

Comment:
`ArgTypes` seems to be auto-generated so we can move them in the region too.

We could also move the `Context` interface to the `story.ts` file since it seems the same for all stories.